monai/bundle/scripts.py

```diff
@@ -247,15 +253,16 @@ def load(
         model_file: the relative path of the model weights or TorchScript module within bundle.
             If `None`, "models/model.pt" or "models/model.ts" will be used.
         load_ts_module: a flag to specify if loading the TorchScript module.
-        bundle_dir: the directory the weights/TorchScript module will be loaded from.
+        bundle_dir: directory the weights/TorchScript module will be loaded from.
             Default is `bundle` subfolder under`torch.hub get_dir()`.
-        source: the place that saved the bundle.
+        source: place that saved the bundle.
```

Comment:
It's not clear what's meant by this, may be "storage location name" and state that "github" is currently the only available value. This should be updated where this argument is used as well.

Reply:
Thanks Eric, I updated the docstrings.